IQKeyboardManager/IQToolbar/IQUIView+IQKeyboardToolbar.h

```diff
@@ -177,7 +177,7 @@
  @param image Image icon to use as right button.
  @param target Target object for selector.
  @param action Right button action name. Usually 'doneAction:(IQBarButtonItem*)item'.
- @param shouldShowPlaceholder A boolean to indicate whether to show textField placeholder on IQToolbar'.
+ @param showPlaceholder A boolean to indicate whether to show textField placeholder on IQToolbar'.
```

Comment:
That makes no sense... If that was the case, your first parameter should be called `addRightButtonOnKeyboardWithImage`. If you want to make it consistent, at least make the name of the variable match with all the other similar methods of the same class. Which style do you prefer? The name after the `@param` tag _must_ match the variable name.
I just noticed this cuz Xcode is throwing documentation warnings in your Cocoapod (and btw, after this merge, if you could release an update to your pod, that'd be awesome :))

Btw no offenses intended, I'm just trying to help 😃